src/app/shared/directives/activate.directive.ts

```diff
@@ -18,6 +18,10 @@ export class ActivateDirective {
   handleActivate(event: KeyboardEvent | MouseEvent) {
 
     if (event instanceof KeyboardEvent) {
+      // if the active element is an input, disregard the event
+      if (document.activeElement.tagName === 'INPUT') {
```

Comment:
Maybe we should have another clause in this statement to check to make sure the input isn't of type 'checkbox', 'button', 'submit', or 'reset'? Those are all inputs that are technically 'activateable', right?

Reply:
I'm going to check for the type of text because the directive won't affect any of the other types.